term-utils/agetty.c

```diff
@@ -2864,12 +2869,23 @@ static void output_special_char(struct issue *ie,
 	case 'U':
 	{
 		int users = 0;
-		struct utmpx *ut;
-		setutxent();
-		while ((ut = getutxent()))
-			if (ut->ut_type == USER_PROCESS)
-				users++;
-		endutxent();
+#ifdef USE_SYSTEMD
+		if (sd_booted() > 0)
```

Comment:
missing bracket? or indentation is wrong for the `if (users < 0`

Reply:
Missing backet, thanks for spotting this.